popup.js

```diff
@@ -0,0 +1,157 @@
+const submitBtn = document.getElementById('submit-btn')
```

Comment:
I added all the variables here - I don't know if this is how it's done or if there is a better and slimmer way to do this.